internal/rbd/rbd_attach.go

```diff
@@ -229,19 +229,19 @@ func createPath(ctx context.Context, volOpt *rbdVolume, cr *util.Credentials) (s
 		mapOptions = append(mapOptions, "--read-only")
 	}
 	// Execute map
-	output, err := execCommand(rbd, mapOptions)
+	stdout, stderr, err := util.ExecCommand(ctx, rbd, mapOptions...)
 	if err != nil {
-		klog.Warningf(util.Log(ctx, "rbd: map error %v, rbd output: %s"), err, string(output))
+		klog.Warningf(util.Log(ctx, "rbd: map error %v, rbd output: %s"), err, stderr)
```

Comment:
it should have been stdout , Isnt it ?

Reply:
interchably using stdout and stderr is confusing..

Reply:
Before, stdout+stderr were combined into a single output. Now there is a split between stdout and stderr. Error messages would be placed in stderr, so that contains the valuable information for troubleshooting.

Reply:
sure. but in above warning we say here is "err:" and here is "output:" , then we use "err" and "stderr" variables, why? . Either using the` stderr `variable in output  is not correct or its a bad naming or the log message is not mentioning it correct.

Reply:
stderr is output, just as stdout? There is no reference to stdout, so I am not sure where the confusion is. There is no output variable anymore.

The message contains `error: %err, output: %stderr`. Do you mean you want to have the variable names instead of error/output?